bindings/python/quokka/program.py

```diff
@@ -440,6 +440,21 @@ def iter_chunk(
                 if chunk.chunk_type in chunk_types:
                     yield chunk
 
+    def read_byte(self, v_addr: AddressT, size: int) -> bytes:
+        """Read raw bytes from a virtual address
+
+        Arguments:
+            v_addr: Virtual address of the data to read
+            size: Size of the data to read
+
+        Returns:
+            The raw data at the specified address
+        """
+
+        if (offset := v_addr - self.base_address) < 0:
```

Comment:
This syntax has been introduced in Python 3.8. As such, we shall enforce the minimal version of Python in the setup.py.